arbitrum-docs/run-arbitrum-node/04-run-local-dev-node.md

```diff
@@ -7,14 +7,138 @@ sidebar_position: 6
 content_type: how-to
 ---
 
-A local Arbitrum Nitro dev node can help you deploy and test smart contracts in a fully controlled environment. This how-to walks you through the process of deploying and running a full development environment on your local machine that includes a Nitro dev node, a dev-mode geth L1, and multiple instances with different roles.
+A local Arbitrum Nitro dev node can help you deploy and test smart contracts in a fully controlled environment. This how-to walks you through the process of deploying and running a full development environment on your local machine that includes a Nitro dev node, a dev-mode geth L1, and multiple instances with different roles. The node is now Stylus-enabled by default, and the setup instructions remain the same as for running a Stylus node.
```

Comment:
```suggestion
A local Arbitrum Nitro dev node can help you deploy and test smart contracts in a fully controlled environment. This how-to walks you through the process of deploying and running a full development environment on your local machine that includes a Nitro dev node, a dev-mode geth L1, and multiple instances with different roles. The node is now Stylus-enabled by default, and the setup instructions remain the same as for running a local Stylus dev node.
```
nitpick to avoid misunderstandings